python/py-precis-i18n/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-precis-i18n
+version             1.0.4
+license             MIT
+supported_archs     noarch
+maintainers         nomaintainer
+
+description         PRECIS framework for Unicode
+long_description    Implementation of PRECIS framework for Unicode to be used with usernames & passwords
+homepage            https://pypi.org/project/precis-i18n
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
```

Comment:
this `master_site` is the default value already set by the `python` PG

Reply:
Sorry, I took an existing portfile (not mine) as a template, and it was there :)
Gonna remove now.